libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/GutenbergEditorFragment.java

```diff
@@ -154,6 +164,14 @@ public void run() {
 
         mEditorFragmentListener.onEditorFragmentInitialized();
 
+        if (getArguments() != null) {
+            mIsNewPost = getArguments().getBoolean(ARG_IS_NEW_POST);
+            if (mIsNewPost) {
+                mTitle.requestFocus();
+                showImplicitKeyboard();
```

Comment:
Is it required the call to `showImplicitKeyboard`? 
I've commented it and didn't notice any difference testing on emulators.

Reply:
Hey @daniloercoli , I checked on mobile device Nexus 5x, 8.1 Android and it can't work without `showImplicitKeyboard `

Reply:
checked here as well on emulator and real device (Pixel 2) and confirmed the keyboard is not shown if that line is removed / commented out:

![withoutshowkeyboard](https://user-images.githubusercontent.com/6597771/50249493-f7583980-03bc-11e9-9890-7f61fdda4dd0.gif)

Reply:
just to double check: tested the same on

- Nexus 4 emulator with Android 5.1.1: 
- Nexus 5 emulator with Android 7.0

Observed the same behavior as above.